Automattic/sensei
# Fix lessons bulk editing Sensei properties

Fixes #2901

#### Changes proposed in this Pull Request:

* JS for handling Lesson bulk edit was not loaded for the page, fix that
* Also move the quick edit JS to this hook
* Fix the Bulk edit submit button selector

#### Testing instructions:

* Select two or more lessons, Bulk Actions > Edit 
* Change the Course or any Sensei-specific property for the selected lessons, click Update
* Changes should be saved


#### Proposed changelog entry for your changes:

* Fix Lessons bulk editing
